autopts/wid/cap.py

```diff
@@ -1041,15 +1044,16 @@ def hdl_wid_20110(params: WIDParams):
         lt1_test_name = params.test_case_name
 
     stack = get_stack()
-    btp.vcp_discover(addr_type, addr)
-    stack.vcp.wait_discovery_completed_ev(addr_type, addr, 10)
 
     if lt1_test_name == 'CAP/COM/CRC/BV-01-C':
         btp.vcp_set_vol(50, addr_type, addr)
+        stack.vcp.wait_vcp_procedure_ev(addr_type, addr, 10)
     if lt1_test_name == 'CAP/COM/CRC/BV-03-C':
         btp.vcp_unmute(addr_type, addr)
+        stack.vcp.wait_vcp_procedure_ev(addr_type, addr, 10)
     if lt1_test_name == 'CAP/COM/CRC/BV-04-C':
         btp.vcp_mute(addr_type, addr)
+        stack.vcp.wait_vcp_procedure_ev(addr_type, addr, 10)
     if lt1_test_name == 'CAP/COM/CRC/BV-05-C':
         btp.vcp_set_vol(13, addr_type, addr)
     if lt1_test_name == 'CAP/COM/CRC/BV-06-C':
```

Comment:
shouldn't we wait for those procedures to complete too?

Reply:
Actually I have just ran these test locally (CAP/COM/CRC/BV-05-C, CAP/COM/CRC/BV-06-C) and there is no wid 20110, so two of these if clauses will not be executed. Should I leave this as is or delete it?
wait for procedure should be added for CAP/COM/CRC/BV-09-C

Reply:
nah, leave is as is, but I'd assume that all vcp procedures are awaited for completion in that wid

Reply:
Added wait_for_ev for all procedures for consistency